doc/classes/Array.xml

```diff
@@ -533,7 +534,7 @@
 			<param index="0" name="func" type="Callable" />
 			<description>
 				Sorts the array using a custom method. The custom method receives two arguments (a pair of elements from the array) and must return either [code]true[/code] or [code]false[/code]. For two elements [code]a[/code] and [code]b[/code], if the given method returns [code]true[/code], element [code]b[/code] will be after element [code]a[/code] in the array.
-				[b]Note:[/b] You cannot randomize the return value as the heapsort algorithm expects a deterministic result. Doing so will result in unexpected behavior.
+				[b]Note:[/b] You cannot randomize the return value as the heapsort algorithm expects a deterministic result. Doing so will result in unexpected behavior. For randomizing the array order you can use [method shuffle].
```

Comment:
Describing how to shuffle in the docs for sort is odd.

Is it because people have tried to use sort as a shuffler? I assumed the use of "randomize" was saying they needed to be consistent and couldn't be arbitrary/random.

Reply:
idk, I thought that if the method is already mentioning randomization, it can as well point to `shuffle()`.